Library/ViewModels/PaymentMethodsViewModel.swift

```diff
@@ -183,10 +185,20 @@ public final class PaymentMethodsViewModel: PaymentMethodsViewModelType,
           .materialize()
       }
 
-    /** TODO: https://kickstarter.atlassian.net/browse/PAY-1954
-     * Add cancellation signal similiar to `shouldCancelPaymentSheetAppearance` in `PledgePaymentMethodsViewModel`
-     */
+    self.shouldCancelPaymentSheetAppearance <~ self.didTapAddCardButtonProperty.signal.ignoreValues()
+      .mapConst(false)
+
+    self.cancelAddNewCardLoadingState = self.shouldCancelPaymentSheetAppearance.signal.filter(isTrue)
+      .ignoreValues()
+    self.shouldCancelPaymentSheetAppearance <~ self.addNewCardSucceededProperty.signal.ignoreValues()
```

Comment:
newbie Reactive question probably, but I see that we're setting `self.shouldCancelPaymentSheetAppearance` twice here. Is there a Reactive way to combine them into a single line? Or is there a benefit to having them split separated like this?